lms/djangoapps/certificates/api.py

```diff
@@ -34,6 +34,15 @@
 log = logging.getLogger("edx.certificate")
 
 
+def is_passing_status(cert_status):
```

Comment:
initially i planned to update that, but that module breaks the encapsulation convention in far more egregious ways (because it predates it) and would need much more work to bring into conformance.  this gets pretty far afield of the intent of the current PR.